src/emqtt_cli.erl

```diff
@@ -298,7 +303,7 @@ parse_cmd_opts([{enable_quic, Enable} | Opts], Acc) ->
     parse_cmd_opts(Opts, [{enable_quic, Enable} | Acc]);
 parse_cmd_opts([{enable_ssl, Enable} | Opts], Acc) ->
     parse_cmd_opts(Opts, [{ssl, Enable} | Acc]);
-parse_cmd_opts([{tls_version, Version} | Opts], Acc) 
+parse_cmd_opts([{tls_version, Version} | Opts], Acc)
```

Comment:
Will we have `Version` as atom here?

Reply:
good catch, probably not. will fix in another PR.